hr_payroll_commission/models/hr_payslip.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# © 2016 Coninckx David (Open Net Sarl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openerp import models, fields, api
+
+
+class HrPayslip(models.Model):
+    _inherit = 'hr.payslip'
+
+    # ---------- Fields management
+
+    invoices = fields.One2many('account.invoice', 'slip_id',
+                               string='Invoices')
+    move_lines = fields.One2many('account.move.line', 'slip_id',
+                                 string='Journal Items')
+
+    # ---------- Utilities
+
+    def _detach_invoices_from_payslip(self):
+        InvoiceObj = self.env['account.invoice']
+        invoices = InvoiceObj.search([('slip_id', 'in', self.ids)])
+        if invoices:
+            invoices.write({'slip_id': False})
+
+    def _detach_move_lines_from_payslip(self):
+        AccountMoveLineObj = self.env['account.move.line']
+        aml = AccountMoveLineObj.search([('slip_id', 'in', self.ids)])
+        if aml:
+            aml.write({'slip_id': False})
+
+    def _attach_invoices_to_payslip(self):
```

Comment:
This should be a get method that is depending on employee_id, and invoices field to have a compute method that should be launched at onchange employee...